pkgs/top-level/python-packages.nix

```diff
@@ -32086,6 +32086,26 @@ EOF
 
   treq = callPackage ../development/python-modules/treq { };
 
+  snakeviz = buildPythonPackage rec {
```

Comment:
This wasn't documented yet. I've [just updated](https://github.com/NixOS/nixpkgs/commit/026cfee6b0dce860841d85b8a50a00e944cf59a2) the Nixpkgs manual to include it